README.md

```diff
@@ -5,76 +5,72 @@ Lightning Flow Scanner (sfdx plug-in)
 
 ## Installation:
 
-npm:
+NPM:
 ```sh-session
 $ npm install -g lightning-flow-scanner
 ```
 
-sfdx:
+SFDX:
 ```sh-session
 $ sfdx plugins:install lightning-flow-scanner
 ```
 
-## Using the scanner:
+## Usage:
 
 ```
-USAGE
-  $ sfdx flow:scan [--json] [--loglevel <level>] [-s] [-u <targetusername>]
+  $ sfdx flow:scan [-c <path>] [-d <directory>] [-e] [-p <path>][-u <targetusername>] [--json] [--loglevel <level>]
+```
 
-OPTIONS
-  --json                                                                            format output as json
+### Options
+```
+  -c, --config <path>                                               provide a path to the configuration file.
 
-  --loglevel=(trace|debug|info|warn|error|fatal)                                    [default: warn] logging level
+  -d, --directory <C:\..\force-app\main\default\flows>              provide a directory to scan.
 
-  -s, --silent                                                                      won't throw errors if violations are found.
+  -e, --throwerrors                                                 set scan to throw an error if a violation is found.
 
-  -u, --targetusername                                                              will retrieve metadata from an org before scan
-```
+  -p, --sourcepath <C:\..\flow1.flow, C:\..\flow2.flow>             provide a comma-separated list of flow paths to scan.
 
-## Optional configurations:
-Use a _.flowscanignore_ file to:
+  -u, --targetusername <username>                                   retrieve the latest metadata from the target before the scan.
 
- - activeRules
- 
- select a limited set of rules to run.
-    
- - overrides
- 
- specify results to ignore. Specify by ruleName and result(if applicable), like shown in the example.
+  --json                                                            set output format as json.
+
+  --loglevel=(trace|debug|info|warn|error|fatal)                    [default: warn] logging level.
 
-#### Example .flowscanignore:
+```
+
+### Configuration file:
+Create a _.flowscanignore_ file to:
```

Comment:
You now have cosmiconfig (from my previous PR), so it won"t be .flowscanignore` but :
- .flow-scanner.yml
- .flow-scanner.json
- etc... ( cf https://github.com/Force-Config-Control/lightning-flow-scanner-sfdx/blob/210100b73947a65c5b0f057a200fd347c459f7ba/src/commands/flow/scan.ts#L145 )

Reply:
interesting, because when I was testing it worked. Ill update the docs to be sure!

Reply:
@RubenHalman I let .flowscanignore as last option to keep ascending compatibility, but it's better to use more classic  config file names :)